maps/chronosphere/main.lua

```diff
@@ -1,15 +1,17 @@
 -- chronosphere --
 
 require "modules.difficulty_vote"
+require "maps.chronosphere.blueprints_vote"

```

Comment:
remove as a whole, no reason to bloat gui, no reason to actually make this changeable or voteable, also it interferes with blueprint function and actually destroys the functionality as of now.

Reply:
Agreed.